File: Core python/Assignment/Assignment 3/q4.py
# Write a program to input all sides of a triangle and check whether triangle is valid or not.

# take side input

x = int(input("enter first side : "))
y = int(input("enter 2nd side : "))
z = int(input("enter 3rd side : "))

# print output base on condition
if((x+y > z)  and (y+z > x) and (x+z > y)):
    print(f'trangle is valid')

else:
    print(f'trangle is not valid')

File: C/Assignments/Assignment 3/q4.c
/* Check the given number is prime or not.
Input: n = 7
Output: Prime  */

void main() {
	int n = 7 ,flag = 1 ;
	
	if(n <= 1) {
		flag = 0 ;
	}
	for(int i = 2 ; i < n ; i++) {
		if(n % i == 0) {
			flag = 1 ;
			break ;
		}
	}
	
	
	if(flag) {
		printf("is prime") ;
	}
	
	else {
		printf("is not prime") ;
	}
}